### ISSUE TITLE
Support for multiple atRules of the same kind

### ISSUE THREAD

**Issue by cvle** [postcss/postcss-js#22]
Can we have support for multiple atRules of the same kind like this?

```js
{
  "@font-face": [
    {
      fontFamily: "'Roboto'",
      fontStyle: "normal",
      fontWeight: 400,
      src: [
        "local('Roboto')",
        "local('Roboto-Regular')",
        `url(${robotoWOFF2}) format("woff2")`,
        `url(${robotoWOFF}) format("woff")`,
      ],
    },
    {
      fontFamily: "'Material Icons'",
      fontStyle: "normal",
      fontWeight: 400,
      src: [
        "local('MaterialIcons')",
        "local('MaterialIcons-Regular')",
        `url(${materialIconFontWOFF2}) format("woff2")`,
        `url(${materialIconFontWOFF}) format("woff")`,
      ],
    },
  ],
};
```

Current workaround is to put spaces after the atRule name:

```js
{
  "@font-face": {
    fontFamily: "'Roboto'",
    fontStyle: "normal",
    fontWeight: 400,
    src: [
      "local('Roboto')",
      "local('Roboto-Regular')",
      `url(${robotoWOFF2}) format("woff2")`,
      `url(${robotoWOFF}) format("woff")`,
    ],
  },
  "@font-face ": {
    fontFamily: "'Material Icons'",
    fontStyle: "normal",
    fontWeight: 400,
    src: [
      "local('MaterialIcons')",
      "local('MaterialIcons-Regular')",
      `url(${materialIconFontWOFF2}) format("woff2")`,
      `url(${materialIconFontWOFF}) format("woff")`,
    ],
  },
};
```

**Comment by ai** [comment]
Sure. Do you want to send PR?

**Comment by cvle** [comment]
Unfortunately I lack the time to do it at the moment and this has low priority.

**Comment by ai** [comment]
Done 5711275

**Comment by ai** [comment]
Released in 0.3

**Comment by cvle** [comment]
Awesome!
